python/hsfs/core/arrow_flight_client.py

```diff
@@ -258,7 +265,12 @@ def _serialize_featuregroup_connector(self, fg, query, on_demand_fg_aliases):
         connector = {}
         if isinstance(fg, feature_group.ExternalFeatureGroup):
             connector["type"] = fg.storage_connector.type
-            connector["options"] = fg.storage_connector.snowflake_connector_options()
+            if fg.storage_connector.type == StorageConnector.SNOWFLAKE:
+                connector[
+                    "options"
+                ] = fg.storage_connector.snowflake_connector_options()
+            elif fg.storage_connector.type == StorageConnector.BIGQUERY:
+                connector["options"] = fg.storage_connector.bigquery_connector_options()
```

Comment:
To avoid this, maybe we could standardize getting the connector_options. by instead having a method named: `fg.storage_connector.connector_options()`

Reply:
I would agree as well, optionally though.

Reply:
changed it, but kept old `snowflake_connector_options ` for backwards compatability